alacritty/src/window_context.rs

```diff
@@ -324,6 +324,9 @@ impl WindowContext {
             return;
         }
 
+        // Force the display to process any pending display update.
+        self.display.process_renderer_update();
```

Comment:
This should always be done even if `!self.dirty`? Wouldn't that cause the back buffer to be locked?